string object_name

---
bool confirmation
string parent
